src/test/java/danielaSCA/TestLayer.java

```diff
@@ -0,0 +1,39 @@
+package danielaSCA;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.filter.log.RequestLoggingFilter;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+public class TestLayer {
+    private final static String BASE_URL = "https://api.todoist.com/rest/v1";
+    private final String token = "2c5090848fc1b7a3d738a7b8f2e5418be887d510";
+
+    public static Response postStatement(String token, Object project) {
+        RestAssured.baseURI = BASE_URL;
+        RequestSpecification request = RestAssured.given().auth().oauth2(token);
+        request.header("Content-Type", "application/json");
```

Comment:
Strings like "application/json" and "/projects" should be in static variables and enums